### PR TITLE
Feature/metadata support

### PR DESCRIPTION
**Pull request by nirpet**

### CODE REVIEW

**Comment by neilcampbell** [comment]
@nirpet Looks like your VS might be setup for tabs again.

**Comment by nirpet** [comment]
Yeah you're right, don't know why. I'll fix it.

**Comment by nirpet** [comment]
Fixed. It happened because of my resharper settings.

File: PactNet.Tests/PactMessage/MessagePactTests.cs [hunk 1]
```diff
@@ -11,233 +11,239 @@
...
+				new ProviderState
+				{
+					Name = "Test state"
+				}
+			};
+			var expectedContent = new
+			{
+				Test = "Test"
+			};
+			var expectedMetdata = new
```
Comment (neilcampbell) [inline PactNet.Tests/PactMessage/MessagePactTests.cs]: This file still appears to be using tabs.

**Pull request merged**